lib/harpy_widgets/theme/harpy_theme.dart

```diff
@@ -23,6 +24,23 @@ const ShapeBorder kDefaultShapeBorder = RoundedRectangleBorder(
 const BorderRadius kDefaultBorderRadius = BorderRadius.all(kDefaultRadius);
 const Radius kDefaultRadius = Radius.circular(16);
 
+/// Fonts
+const List<String> kAssetFonts = [
+  kDefaultBodyFontFamily,
+  kDefaultDisplayFontFamily,
+];
+
+bool isAssetFontFamily(String font) {
+  return kAssetFonts.contains(font);
+}
```

Comment:
```suggestion
```

If we only use it once, we might as well remove it and just call `kAssetFonts.contains` where we need it